<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>440</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>true</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="main_title">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Student Grade Checker</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="search_student">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_student">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Student</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subject_regeisteration">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Subject Registration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subjects_calculations">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Subjects Calculation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subjects_calculations_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Subject</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>20</width>
      <height>301</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="student_services_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Student Services</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="subject_services_title">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Subject Services</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_student">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Student</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
